src/internal/mod.rs

```diff
@@ -147,6 +151,24 @@ pub fn validate_id(id: &str, id_type: &str) -> Result<String, IronOxideErr> {
     }
 }
 
+pub fn validate_simple_policy_id(id: &str, id_type: &str) -> Result<String, IronOxideErr> {
```

Comment:
Maybe `validate_simple_policy_field(field, field_type)` or something? They aren't really IDs in the way we usually talk about them.

Reply:
`validate_simple_policy_field_id` seem okay?

Reply:
Somewhat, sure. I'd still prefer to remove the `_id` there, but I'm not going to argue if you want to keep it.